pkg/teams/teams.go

```diff
@@ -55,29 +60,20 @@ func readTeamMap(filePath string) (teams Teams, err error) {
 		}
 	}()
 
-	err = json.NewDecoder(f).Decode(&teams)
+	decoder := json.NewDecoder(f)
+	decoder.DisallowUnknownFields()
+	err = decoder.Decode(&teams)
 
 	return
 }
 
-// accountsToTeamNames returns a map of accounts to team names
-func (t *Teams) accountsToTeamNames() map[string]string {
-	var a = make(map[string]string)
+// accountsToTeamNames returns a map of Accounts to team names
+func (t *Teams) accountsToTeamNames() map[Account]string {
+	var a = make(map[Account]string)
 	for _, team := range t.Teams {
 		for _, account := range team.Accounts {
 			a[account] = team.Name
```

Comment:
I know this is not new to this PR, but if it were new I would suggest verifying that there aren't duplicate entries for accounts, instead of silently dropping later entries